src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -86,5 +86,10 @@
   },
   "SearchAPI": {
     "SearchAPIEnabled": false
+  },
+  "AzureAD": {
```

Comment:
how about renaming this to something more apt like 'AccessControlSettings' or 'SecuritySettings', something like that.